examples/go-tracepoint-counter/kubernetes-deployment/go-tracepoint-counter-bytecode.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: bpfd.io/v1alpha1
+kind: BpfProgramConfig
+metadata:
+  labels:
+    app.kubernetes.io/name: bpfprogramconfig
+  name: go-tracepoint-counter-example
+spec:
+  ## Must correspond to image section name
+  name: stats
+  type: TRACEPOINT
+  # Select all nodes
+  nodeselector: {}
+  attachpoint:
+    networkmultiattach:
```

Comment:
This will be a `networksingleattach` :) see https://github.com/redhat-et/bpfd/blob/main/bpfd-operator/config/samples/bpfd.io_v1alpha1_tracepoint_bpfprogramconfig.yaml

Reply:
Thank you, it was just copy/paste from TC I hadn't gotten around to yet, as I haven't tested this with BPFD yet (only tested locally at this point during draft). Will update soon in upcoming cycles :+1: